dc/council/code/titles/28/sections/28-4004.xml

```diff
@@ -26,11 +26,13 @@
     <num>(d)</num>
     <text>If a prospective hearing aid user has a bona fide religious belief which precludes him or her from having a medical examination as required in section 28-4004(a), the prospective hearing aid user may waive the medical examination requirement; provided, that the prospective hearing aid user signs the following statement, printed in ten (10)-point type:</text>
     <text>“My religious beliefs require that I waive the medical examination and the hearing aid evaluation required by the ‘Hearing Aid Dealer and Consumers Act of 1977’ for the purchase of a hearing aid. I voluntarily waive the medical examination, notwithstanding the fact that I have been advised by</text>
-    <para>
-      <heading>HEARING AID DISPENSER’S NAME</heading>
-      <text>that my best health interest would be served if I had a medical evaluation by a physician who is an ear specialist.”</text>
-      <text>No registrant shall seek to induce a prospective hearing aid user to execute such a waiver.</text>
-    </para>
+    <text>
+      <center>
```

Comment:
I think @alexmorken might be referring to the fact that the heading in 28-3863 didn't have a `<strong>`, but this one does. (Division V) I would note that in both cases, these are headings for "includes". I would also note that in the word document the two items are in fact formatted differently.